backend/src/v2/compiler/argocompiler/testdata/create_mount_delete_dynamic_pvc.yaml

```diff
@@ -8,17 +8,17 @@ spec:
     parameters:
     - name: kubernetes-comp-comp
       value: '{"pvcMount":[{"mountPath":"/data","taskOutputParameter":{"outputParameterKey":"name","producerTask":"createpvc"}}]}'
-    - name: components-95f802401136aebf1bf728a6675d7adba5513b53673a3698e00a6d8744638080
+    - name: components-b34273359995b3746ecf1bb58ac4bd6c54d47b6fdc35b013bb7962946f322a19
```

Comment:
similar to [this](https://github.com/kubeflow/pipelines/pull/11159/files#r1747562059) maybe adding your steps on how you generated these would be useful in the future.

Reply:
The test will fail and the output will be that you should update to the value.